operatorcourier/nest.py

```diff
@@ -58,18 +71,23 @@ def nest_bundles(yaml_files, registry_dir, temp_registry_dir):
             yaml.dump(csv, outfile, default_flow_style=False)
             outfile.flush()
 
-        csv_crds = csv["spec"]["customresourcedefinitions"]["owned"]
-        for csv_crd in csv_crds:
-            crd_name = csv_crd["name"]
-            if crd_name in crds:
-                crd = crds[crd_name]
-                crdfile_name = os.path.join(csv_folder, '%s.crd.yaml' % crd_name)
-                with open(crdfile_name, 'w') as outfile:
-                    yaml.dump(crd, outfile, default_flow_style=False)
-                    outfile.flush()
-            else:
-                errors.append("CRD %s mentioned in CSV %s was not found in directory."
-                              % (crd_name, csv_name))
+        if "customresourcedefinitions" in csv["spec"]:
```

Comment:
`KeyError` here if `spec` is not defined?

Reply:
I append an error earlier on for this, but it's true that this would throw a `KeyError`
Should I break early if a key is missing, or should I continue to aggregate errors?

Reply:
I'm leaning towards using `continue` whenever a required key isn't found